### PR TITLE
Readme

### PR DESCRIPTION
**Pull request by 9728627**
update the readme file 
introduce the awesome fetatures
(50 xp)

### CODE REVIEW

File: src/edu/ucsb/cs56/projects/games/fish_animation/Menu.java [hunk 1]
```diff
@@ -92,7 +92,7 @@ public void makegui() {
 			PauseBGM.setIcon(new ImageIcon(imgPause));
 		} catch (IOException ex) {
 		}
-
+
```
Comment (xavierholt) [inline src/edu/ucsb/cs56/projects/games/fish_animation/Menu.java]: This change shouldn't be part of this PR.

Comment (9728627) [reply src/edu/ucsb/cs56/projects/games/fish_animation/Menu.java]: No, it shouldn't. I think I just accidently created new line? I deleted it again. I do not think it will affect the other codes.

Comment (xavierholt) [reply src/edu/ucsb/cs56/projects/games/fish_animation/Menu.java]: Sounds good.  You're right that it won't cause any problems.  I just like to keep the Git history clean.  If you see random files changed as part of PRs that should be unrelated you get confused.

Comment (9728627) [reply src/edu/ucsb/cs56/projects/games/fish_animation/Menu.java]: so do I need to do anything before you merge this PR

Comment (xavierholt) [reply src/edu/ucsb/cs56/projects/games/fish_animation/Menu.java]: Yeah.  You need to get rid of this change.

File: README.md [hunk 2]
```diff
@@ -13,12 +13,34 @@ YES | mastergberry | jovo123, jcryan | A fish animation
...
-Currently the game runs, and we have the following classes:
+### Currently the game runs, and we have the following classes:
 * Game logics: Score, Score Manager, ScoreComparer, ScumOftheSea, JellyFish, Fish, Shark, SoundEffects, Character.
 * GUI: Menu, GeneralPathWrapper.
 * Mixed: FishAnimationEnvironment.
 
 
 The main function to start the program is in Menu.java. Menu.java takes care of building the main menu of the game, and some Jbuttons on it are tied to some functions in FishAnimationEnvironment.java, where the bulk of the game is defined. Currently, FishAnimationEnvironemnt.java is a mix of Game Logics and GUI stuffs, so it is pretty hard to read through and understand. A code refactoring on this java file is highly recommanded: try to separate the game logic from GUI. The Score Class implements Serializable so that the scores can be stored and provides that game-stop feature, but it is more usual to record the highscores in a database. It is challanging, but a nice chance to learn how to drive the database inside a Java program using JDBC. 
+
+# F17 update
```
Comment (xavierholt) [inline README.md]: Your comments are a better intro to the game than the "F16 Final Remarks" - move your stuff to the top!

File: README.md [hunk 3]
```diff
@@ -13,12 +13,34 @@ YES | mastergberry | jovo123, jcryan | A fish animation
...
+* There's no time limit, but watch out! Jellyfish are out to get you!
+* If you eat the jellyfish, you'll lose health!
+* If your health is 0 or you get too many negative points, you lose.
+* There are three different difficulties, so test your skills on all of them! Good eating!
+
+
+### How to Play
+* To eat the fish
+* click on the shark and hold down the mouse button to move your shark's mouth to the fish.
+* You can also use the arrow keys to move the fish as well."
```
Comment (xavierholt) [inline README.md]: - This should be "shark," not "fish."
 - There's a stray `"` at the end of this one.
 - The next four lines are kind of obvious.  Let's remove them.

Comment (9728627) [reply README.md]: fixed

File: README.md (hunk 2 above)
Comment (9728627) [reply README.md]: cool

File: src/edu/ucsb/cs56/projects/games/fish_animation/Menu.java (hunk 1 above)
Comment (9728627) [reply src/edu/ucsb/cs56/projects/games/fish_animation/Menu.java]: fixed

**Comment by xavierholt** [comment]
Looks good!  I still see a difference in that one Java file, but meh.  Whatever.  Merging!

Fixes https://github.com/UCSB-CS56-Projects/cs56-games-fish-animation/issues/44.

+50 XP
